Library/Formula/websocketd.rb

```diff
@@ -0,0 +1,24 @@
+class Websocketd < Formula
+  homepage "http://websocketd.com"
+  url "https://github.com/joewalnes/websocketd/archive/v0.2.10.tar.gz"
+  sha1 "4387c39aaf77e1bb5b6991358f02611a7b5dac1e"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p "src/github.com/joewalnes"
+    ln_s "../../../", "src/github.com/joewalnes/websocketd"
+
+    system "go", "get", "./src/github.com/joewalnes/websocketd"
```

Comment:
Sorry just noticed this: does this use any other Go modules? If so, can you use the `go_resource` pattern (`grep` other formulae for usage).